docs/docs/component-css.md

```diff
@@ -2,7 +2,192 @@
+  display: flex;
+  align-items: center;
+  margin: 0 auto 12px auto;
+}
+
+.user:last-child {
+  margin-bottom: 0;
+}
+
+.avatar {
+  flex: 0 0 96px;
+  width: 96px;
+  height: 96px;
+  margin: 0;
+}
+
+.description {
+  flex: 1;
+  margin-left: 18px;
+  padding: 12px;
+}
+
+.username {
+  margin: 0 0 12px 0;
+  padding: 0;
+}
+
+.excerpt {
+  margin: 0;
+}
+```
+
+Now import that file into the `about-css-modules.js` page you created earlier, by adding the following on lines 2 and 3.
+(The `console.log(styles)` code logs the resulting import so you can see what the processed file looks like).
+
+```javascript:title=src/pages/about-css-modules.js
+import styles from "./about-css-modules.module.css"
+console.log(styles)
+```
+
+If you open the developer console (using e.g. Firefox or Chrome's developer tools) in your browser, you'll see:
+
+![css-modules-console](./images/css-modules-console.png)
+
+If you compare that to your CSS file, you'll see that each class is now a key in
+the imported object pointing to a long string e.g. `avatar` points to
+`about-css-modules-module---avatar----hYcv`. These are the class names CSS
+Modules generates. They're guaranteed to be unique across your site. And because
+you have to import them to use the classes, there's never any question about
+where some CSS is being used.
+
+Use your styles to create a `User` component.
+
+Now, create the new component inline in the `about-css-modules.js` page
+component. The general rule of thumb is this: if you use a component in multiple
+places on a site, it should be in its own module file in the `components`
+directory. But, if it's used only in one file, create it inline.
+
+Modify `about-css-modules.js` so it looks like the following:
+
+```jsx{7-19,25-34}:title=src/pages/about-css-modules.js
+import React from "react"
+import styles from "./about-css-modules.module.css"
+import Container from "../components/container"
+
+console.log(styles)
+
+const User = props => (
+  <div className={styles.user}>
+    <img src={props.avatar} className={styles.avatar} alt="" />
+    <div className={styles.description}>
+      <h2 className={styles.username}>{props.username}</h2>
+      <p className={styles.excerpt}>{props.excerpt}</p>
+    </div>
+  </div>
+)
+
+export default () => (
+  <Container>
+    <h1>About CSS Modules</h1>
+    <p>CSS Modules are cool</p>
+    <User
+      username="Jane Doe"
+      avatar="https://s3.amazonaws.com/uifaces/faces/twitter/adellecharles/128.jpg"
+      excerpt="I'm Jane Doe. Lorem ipsum dolor sit amet, consectetur adipisicing elit."
+    />
+    <User
+      username="Bob Smith"
+      avatar="https://s3.amazonaws.com/uifaces/faces/twitter/vladarbatov/128.jpg"
+      excerpt="I'm Bob smith, a vertically aligned type of guy. Lorem ipsum dolor sit amet, consectetur adipisicing elit."
+    />
+  </Container>
+)
+```
+
+The finished page should now look like:
+
+![css-modules-final](./images/css-modules-final.png)
+
+### Other CSS options
```

Comment:
Let's keep this section here though, and feel free to add anything else to the list that you'd like!

Reply:
Is this the code example you'd like me to keep? I can change the wording to make more sense in the current environment.

Reply:
@amberleyromo @shannonbux